lobbyheads-api/src/main/java/com/eternalcode/lobbyheads/head/event/HeadUpdateEvent.java

```diff
@@ -0,0 +1,40 @@
+package com.eternalcode.lobbyheads.head.event;
+
+import java.util.UUID;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * This event is called when a head is updated.
+ **/
+public class HeadUpdateEvent extends Event {
+
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+
+    private final UUID playerUniqueId;
```

Comment:
I think this name is nonsense, I would just change it to uniqueId as it is in bukkit-api

Reply:
rollczi